Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:14.15.1 AS build
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+ARG react_app_sockets=http://localhost:8000
```

Comment:
better to name those args: socket_address (without react_app) as this naming is only relevant when passing them into the Create React App.

Reply:
Ok, I will change that.

Reply:
But regarding this, should the env variable name also change, because I put that name considering the env variable name and is only to pass an env variable as an argument to Docker in the build process.